internal/theia/pkcs7.go

```diff
@@ -0,0 +1,30 @@
+package theia
+
+import (
+	"encoding/pem"
+	"fmt"
+
+	theia "github.com/IBM/cbomkit-theia/scanner/x509"
+	"github.com/smallstep/pkcs7"
```

Comment:
Why we need dependency on smallstep and cannot use standard golang.org/x/crypto/pkcs7 or something similar?